topics/dev/images/core_files_ci.plantuml.txt

```diff
@@ -0,0 +1,25 @@
+@startmindmap
+!include plantuml_style.txt
+!include plantuml_options.txt
+
+* /
+**:<b><i>.ci</i></b>
+ scripts for linting Python, etc..;
+***:<b><i>jenkins/</i></b>
+ sub-directory per Jenkins job type, wrappers
+ around run_tests for Dockerized testing
+ can be used outside Jenkins;
+**** <b><i>api/</i></b>
+**** <b><i>integration/</i></b>
+**** <b><i>selenium/</i></b>
+*** <b><i>.circleci/</i></b>
```

Comment:
Both `.circleci` and `.github/workflows` should be children of `/`, not of `.ci/`

Reply:
Rebased with a fix I believe.